recipes/cdsdashboards/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cdsdashboards" %}
+{% set version = "0.0.19" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ccc6eebb9e5471e6dcef8bc7d028561a8c8649b5d7b8dfcb3472ba3796fe7867
+
+build:
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+      script: "python -m pip install . --no-deps --ignore-installed -vv"
```

Comment:
This should use `{{ PYTHON }}`

Reply:
Thanks for the review @scopatz 
The suggestion does not work. I tried it on https://github.com/conda-forge/staged-recipes/pull/11280.

In the mentioned PR, the work around was to use external script or I keep this syntax (as done in [jupyterhub-feedstock](https://github.com/conda-forge/jupyterhub-feedstock/blob/master/recipe/meta.yaml#L25)) as it is simpler than having two identical script (except for the os environment name).

Reply:
I think this is picking up the wrong python then.  It should at least be `$PREFIX/bin/python`

Reply:
Thanks for the info. So I switched to external script to use $PYTHON/%PYTHON%.